core/templates/pages/library-page/library-page.component.ts

```diff
@@ -316,38 +332,40 @@ export class LibraryPageComponent {
                     let ACTIVITY_TYPE_EXPLORATION = 'exploration';
                     let ACTIVITY_TYPE_COLLECTION = 'collection';
 
-                    activitySummaryDicts.forEach((activitySummaryDict) => {
-                      if (activitySummaryDict.activity_type === (
-                        ACTIVITY_TYPE_EXPLORATION)) {
-                        this.activitiesOwned.explorations[
-                          activitySummaryDict.id] = false;
-                      } else if (activitySummaryDict.activity_type === (
-                        ACTIVITY_TYPE_COLLECTION)) {
-                        this.activitiesOwned.collections[
-                          activitySummaryDict.id] = false;
-                      } else {
-                        this.loggerService.error(
-                          'INVALID ACTIVITY TYPE: Activity' +
+                    activitySummaryDicts.forEach(
+                      (activitySummaryDict) => {
+                        if (activitySummaryDict.activity_type === (
+                          ACTIVITY_TYPE_EXPLORATION)) {
+                          this.activitiesOwned.explorations[
+                            activitySummaryDict.id] = false;
+                        } else if (activitySummaryDict.activity_type === (
+                          ACTIVITY_TYPE_COLLECTION)) {
+                          this.activitiesOwned.collections[
+                            activitySummaryDict.id] = false;
+                        } else {
+                          this.loggerService.error(
+                            'INVALID ACTIVITY TYPE: Activity' +
                           '(id: ' + activitySummaryDict.id +
                           ', name: ' + activitySummaryDict.title +
                           ', type: ' + activitySummaryDict.activity_type +
                           ') has an invalid activity type, which could ' +
                           'not be recorded as an exploration or a ' +
                           'collection.'
-                        );
-                      }
-                    });
+                          );
+                        }
+                      });
 
                     response.explorations_list.forEach(
                       (ownedExplorations) => {
                         this.activitiesOwned.explorations[
                           ownedExplorations.id] = true;
                       });
 
-                    response.collections_list.forEach((ownedCollections) => {
-                      this.activitiesOwned.collections[
-                        ownedCollections.id] = true;
-                    });
+                    response.collections_list.forEach(
+                      (ownedCollections) => {
+                        this.activitiesOwned.collections[
+                          ownedCollections.id] = true;
+                      });
```

Comment:
Why is this modified?

Reply:
This was modified by mistake. Reverted the change.